mono/metadata/custom-attrs.c

```diff
@@ -823,18 +826,12 @@ create_custom_attr (MonoImage *image, MonoMethod *method, const guchar *data, gu
 	}
 
 	named = p;
-	attr = mono_object_new_checked (mono_domain_get (), method->klass, error);
+	attr = mono_object_new_handle (mono_domain_get (), method->klass, error);
 	goto_if_nok (error, fail);
 
-	MonoObject *exc = NULL;
-	mono_runtime_try_invoke (method, attr, params, &exc, error);
+	(void)mono_runtime_try_invoke_handle (method, attr, params, error);
```

Comment:
I'm a little nervous about this transform, that I recently started. Maybe restore the exception parameter. So far I *believe* it is equivalent, and we can how that holds up upon further conversion. Everyone checks error and then exc/set_exception_instance, so set_set_exception_instance in invoke. I didn't survey all users, just the ones I have changed.

Reply:
Yea the form with both `exc` and `error` was useful while we still had places where we called invoke but didn't have a way of threading `error` out, while `exc` was used in some manner.  We're probably in a better place now.